src/Stl/Generators/RandomStringGenerator.cs

```diff
@@ -67,5 +71,31 @@ public string Next(int length, string? alphabet = null)
                 buffer.Release();
             }
         }
+#else
+        public string Next(int length, string? alphabet = null)
+        {
+            if (alphabet == null)
+                alphabet = Alphabet;
+            else if (alphabet.Length < 1)
+                throw new ArgumentOutOfRangeException(nameof(alphabet));
+            var byteBuffer = new byte[length];
```

Comment:
Think if a new String.Create(Span<char>) would help to keep the only version of this class too.

Reply:
.netstandard 2.0 does not support new String.Create(Span)
I did some optimizations with array pooling.